diff --git a/telegram-bot-api/Client.cpp b/telegram-bot-api/Client.cpp
index 86bd10d..350ccf9 100644
--- a/telegram-bot-api/Client.cpp
+++ b/telegram-bot-api/Client.cpp
@@ -4277,6 +4277,8 @@ class Client::JsonStickerSet final : public td::Jsonable {
     object("sticker_type", type);
     object("contains_masks", td::JsonBool(type == "mask"));
 
+    object("__custom__id", std::to_string(sticker_set_->id_));
+
     object("stickers", JsonStickers(sticker_set_->stickers_, client_));
   }
 
